### PR TITLE
.deb & .appimage package build 

### PR DESCRIPTION
**Pull request by ZetaoYang**
using `dpkg -b` to build .deb and not recommended to use `dpkg-buildpackage`
suggest to remove the `debian` dir.
all package building in `package` dir.
the .deb test on Ubuntu16.04 successfully.
Now, you can release .deb and AppImage in Github Release pages.

### CODE REVIEW

File: package/deb/DEBIAN/control [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Package: flameshot
+Version: 0.5.0
+Architecture: amd64
+Maintainer: Ahmed Zetao Yang <vitzys@outlook.com>
+Installed-Size: 54400
+Depends: libstdc++6, libc6 (>= 2.14), libgcc1, libglib2.0-0, 	libglib2.0-0, libx11-6, libpng12-0, libharfbuzz0b, zlib1g, libgl1-mesa-glx | libgl1, libdbus-1-3, libproxy1v5, libpcre16-3, libffi6, libpcre3, libxcb1, libfreetype6, libgraphite2-3, libxext6, libxau6, libxdmcp6, libselinux1, liblzma5, libgcrypt20, libgpg-error0, libsystemd0, libxcb-xinerama0
```
Comment (lupoDharkael) [inline package/deb/DEBIAN/control]: Why al these dependencies instead using the specific ones of QT? isn't this harder to maintain? I don't think it's a good idea.

File: package/appimage/flameshot.AppDir/flameshot.desktop [hunk 2]
```diff
@@ -0,0 +1,11 @@
+[Desktop Entry]
```
Comment (lupoDharkael) [inline package/appimage/flameshot.AppDir/flameshot.desktop]: Flameshot already maintains desktop files on the ./docs/desktopEntry directory, why duplicate the efforts?

File: package/README.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+##package build
+
+### deb
+
+Use `dpkg -b deb_dir flameshot_amd64.deb` to build .deb package
+
+### appimage
+
+Use [linuxdeployqt](https://github.com/probonopd/linuxdeployqt) to copy libraries and plugins that the `flameshot` uses, and generates an [AppImage](https://github.com/AppImage/AppImageKit).
```
Comment (lupoDharkael) [inline package/README.md]: Fix: that `flameshot` uses, and generate an

File: package/deb/usr/share/applications/flameshot.desktop [hunk 4]
```diff
@@ -0,0 +1,17 @@
+[Desktop Entry]
```
Comment (lupoDharkael) [inline package/deb/usr/share/applications/flameshot.desktop]: Same as in the other desktop file.

**Comment by lupoDharkael** [comment]
What I same about the desktop files applies for the icons too. The ./img directory contains an icon and if I need to change or update it it should be as easy as just change it in one place and not in 4 or 5.

**Comment by ZetaoYang** [comment]
You are rignt,I will fix it.

**Comment by lupoDharkael** [comment]
I think I'm going to generalize the desktop files so they can be used with any path. The only problem is org.dharkael.Flameshot.service which requires the path to the binary. I'm going to create a script to generate the file for an arbitrary path.

With these changes you don't need desktopfile generation scripts.

**Comment by lupoDharkael** [comment]
https://github.com/lupoDharkael/flameshot/commit/c7147f913197318a1438d0c66d3380f68c991280

Ok here it is, hope it helps.

**Comment by ZetaoYang** [comment]
you forgot `;` of `Categories=Graphics;`.

**Comment by lupoDharkael** [comment]
Fixed

**Comment by ZetaoYang** [comment]
there are multi `name` field in `docs/desktopEntry/package/flameshot.desktop`

**Comment by lupoDharkael** [comment]
https://github.com/lupoDharkael/flameshot/commit/dc96f99281e2e4ae2b226616127b46c98bf26708

**Comment by ZetaoYang** [comment]
https://travis-ci.org/ZetaoYang/flameshot/builds/321746240
When `dpkg-buildpackage -rfakeroot -B`, here it went wrong:  
make[1]: *** [controller.o] Error 1
make[1]: Leaving directory `/home/travis/build/ZetaoYang/flameshot'
dh_auto_build: make -j1 returned exit code 2
make: *** [build-arch] Error 25
dpkg-buildpackage: error: debian/rules build-arch gave error exit status 2

**Pull request closed without merging**